### ISSUE TITLE
Build a version using Google Workspace Add-ons?

### ISSUE THREAD

**Issue by TylerFisher** [news-catalyst/google-app-scripts#169]
[Google Workspace add-ons](https://developers.google.com/gsuite/add-ons/how-tos/building-gsuite-addons) are Google's new add-on type, and it seems like what they'll be supporting and improving going forward. They offer a better UI through their Card stack and are the only way to test and deploy with the new version of their editor.

I've played around with the new add-ons and it seems we could do everything our current add-on does. Rewriting this would mean rewriting the UI to use the new card stack approach, and attaching all of the events to those cards. The underlying functionality shouldn't have to change at all.

**Comment by TylerFisher** [comment]
I think the lack of viable multi-select options is a real killer here for the time being.

**Comment by jacqui** [comment]
@TylerFisher yeah I think that's the right call. I see they suggested using a checkbox group. Not really what we need.

For reference, here's a link to the issue you logged with Google: https://issuetracker.google.com/issues/177345652

**Comment by chelsea-tnc** [comment]
Recently added a comment to the issue Tyler logged with Google. I checked the library myself and didn't see any UI Kit updates that would be of use for this. Marking as low priority for now.

**Comment by TylerFisher** [comment]
I would definitely consider this low-priority. As far as I can tell, editor add-ons (what we use) aren't being deprecated or anything.
